### PR TITLE
CORE-14632: Update flow driver tests.

### PR DESCRIPTION
**Pull request by chrisr3**
Update flow driver tests for latest versions.

### CODE REVIEW

**Review by khutsijabari — DISMISSED** [review]

File: workflows/src/test/java/com/r3/developers/apples/workflows/ApplesFlowDriverTests.java [hunk 1]
```diff
@@ -38,27 +38,29 @@ public class ApplesFlowDriverTests {
...
-                        .forEach(it -> vNodes.put(it.getHoldingIdentity().getX500Name(), it))
-        );
+        vNodes = driver.let(dsl -> {
+            dsl.startNodes(Set.of(alice, bob));
+            return dsl.nodesFor("workflows");
+        });
 
-        if (vNodes.isEmpty()) fail("Failed to populate vNodes");
+        assertThat(vNodes)
+            .withFailMessage("Failed to populate vNodes")
```
Comment (tlawson3) [inline workflows/src/test/java/com/r3/developers/apples/workflows/ApplesFlowDriverTests.java:55]: same update on this one please

File: workflows/src/test/java/com/r3/developers/apples/workflows/ApplesFlowDriverTests.java [hunk 2]
```diff
@@ -38,27 +38,29 @@ public class ApplesFlowDriverTests {
...
+            .isNotEmpty();
     }
 
     @Test
     void test_that_CreateAndIssueAppleStampFlow_returns_correct_message() {
         UUID stampId = createAndIssueAppleStamp("Stamp # 0001", bob, alice);
         logger.info("result: {}", stampId);
-        assertThat(stampId).isNotNull();
-        assertThat(stampId.toString()).hasSize(36); // standard UUID length
+        assertThat(stampId).isInstanceOf(UUID.class);
```
Comment (tlawson3) [inline workflows/src/test/java/com/r3/developers/apples/workflows/ApplesFlowDriverTests.java:63]: 🚀

File: workflows/src/test/java/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowDriverTests.java [hunk 3]
```diff
@@ -11,14 +11,17 @@
...
-import java.util.*;
+import java.util.Map;
+import java.util.Set;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.fail;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
 
-public class MyFirstFlowDriverTests {
+@TestInstance(PER_CLASS)
```
Comment (tlawson3) [inline workflows/src/test/java/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowDriverTests.java:23]: Why do we need this when we're using an EachTestDriver?

**Review by tlawson3 — CHANGES_REQUESTED** [review]

**Review by khutsijabari — COMMENTED** [review]
Are we expecting CorDapp developers with existing CorDapps to make the same changes in the gradle-wrapper.properties to be able to use Flow Testing Driver?

File: workflows/src/test/java/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowDriverTests.java (hunk 3 above)
Comment (chrisr3) [reply workflows/src/test/java/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowDriverTests.java:23]: I suppose we don't _need_ to... unless we also want to make `ObjectMapper` non-static?

Comment (tlawson3) [reply workflows/src/test/java/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowDriverTests.java:23]: Ok leave it as we'll keep ObjectMapper static

**Review by tlawson3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @chrisr3

**Pull request merged**